src/number.ts

```diff
@@ -0,0 +1,243 @@
+	return getGlobalize(locale).currencyFormatter(currency, normalizeOptions(options));
+}
+
+/**
+ * Return a function that formats a number according to the specified configuration and or locale.
+ *
+ * @param options
+ * An optional configuration of settings that determine how the number string will be formatted.
+ *
+ * @param locale
+ * An optional locale. Defaults to the root locale.
+ *
+ * @return
+ * A function that accepts a number and returns a formatted string.
+ */
+export function getNumberFormatter(options?: NumberFormatterOptions, locale?: string): (value: number) => string {
+	return getGlobalize(locale).numberFormatter(normalizeOptions(options));
+}
+
+/**
+ * Parse a number from a string based on the provided configuration and or locale.
+ *
+ * @param options
+ * An optional config that describes the format of the string.
+ *
+ * @param locale
+ * An optional locale. Defaults to the root locale.
+ *
+ * @return
+ * The parsed number.
+ */
+export function getNumberParser(options?: NumberFormatterOptions, locale?: string): (value: string) => number {
+	return getGlobalize(locale).numberParser(normalizeOptions(options));
+}
+
+/**
+ * Return a function that accepts a number and returns that number's plural group.
+ *
+ * @param options
+ * An optional configuration that determines whether the numerical value should be treated as a cardinal
+ * or ordinal number.
+ *
+ * @param locale
+ * An optional locale. Defaults to the root locale.
+ *
+ * @return
+ * A function that accepts a number and returns the corresponding plural group.
+ */
+export function getPluralGenerator(options?: PluralGeneratorOptions, locale?: string): (value: number) => PluralGroup {
+	return getGlobalize(locale).pluralGenerator(normalizeOptions(options)) as (value: number) => PluralGroup;
+}
+
+/**
+ * Return a function that parses a number from a string based on the provided configuration and or locale.
+ *
+ * @param value
+ * The string to parse.
+ *
+ * @param options
+ * An optional config that describes the format of the string.
+ *
+ * @param locale
+ * An optional locale. Defaults to the root locale.
+ *
+ * @return
+ * A function that accepts a string and returns a number.
+ */
+export function parseNumber(value: string, options?: NumberFormatterOptions, locale?: string): number {
+	return getGlobalize(locale).parseNumber(value, normalizeOptions(options));
+}
+
+/**
+ * Return the plural group from a number.
+ *
+ * @param value
+ * The number.
+ *
+ * @param options
+ * An optional configuration that determines whether the numerical value should be treated as a cardinal
+ * or ordinal number.
+ *
+ * @param locale
+ * An optional locale. Defaults to the root locale.
+ *
+ * @return
+ * The plural group.
+ */
+export function pluralize(value: number, options?: PluralGeneratorOptions, locale?: string): PluralGroup {
+	return getGlobalize(locale).plural(value, normalizeOptions(options)) as PluralGroup;
```

Comment:
The `Globalize.js` typings specify the return value of `plural` as a string; I've just narrowed it to the specific values that are actually returned. I can update the Globalize typing and open a PR with DefinitelyTyped, or just change this to `string`. I'm inclined to go the latter route for the time being, and then change it back to the more specific `PluralGroup` if the PR is accepted.